# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/base/content/browser.js
# Commit: 8287c3d0c106
# Full Hash: 8287c3d0c1064096ee7c162060e6615d72722ee0
# Author: Christoph Walcher <christoph-wa@gmx.de>
# Date: 2019-06-20 09:46:31
# Regressor Bug: 909760
# File Overlap Count: 1
# Description:
#   Bug 909760 - Show download progress in the MacOS Finder r=spohl,mak
#   
#   Show download progress in the MacOS Finder
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30688
# ==============================================================================

diff -r 5b9a3de04646 -r 8287c3d0c106 browser/base/content/browser.js
--- a/browser/base/content/browser.js	Wed Jun 19 18:39:02 2019 +0000
+++ b/browser/base/content/browser.js	Wed Jun 19 18:48:49 2019 +0000
@@ -1946,6 +1946,10 @@
         DownloadsCommon.initializeAllDataLinks();
         ChromeUtils.import("resource:///modules/DownloadsTaskbar.jsm", {})
           .DownloadsTaskbar.registerIndicator(window);
+        if (AppConstants.platform == "macosx") {
+          ChromeUtils.import("resource:///modules/DownloadsMacFinderProgress.jsm")
+            .DownloadsMacFinderProgress.register();
+        }
       } catch (ex) {
         Cu.reportError(ex);
       }